mooncake-transfer-engine/src/transport/rdma_transport/rdma_context.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include "transport/rdma_transport/rdma_context.h"
 
+#include <cuda.h>
```

Comment:
By default, the CUDA library is not required. It should be included using #ifdef...#endif.

Reply:
Done